méli-mélo/th-empathy/empathy.html

```diff
@@ -0,0 +1,92 @@
+---
+title: Empathy theme
+dateModified: 2024-04-30
+description: Empathy CSS styles
+lang: en
+pageclass: cnt-wdth-lmtd
+css:
+- style.css
+- https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/méli-mélo/2025-04-nahanni.css
+script:
+- https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/méli-mélo/2025-04-nahanni.js
+layout: without-h1
+---
+<h1 property="name" id="wb-cont" dir="ltr" class="empathy">{{ page.title }}</h1>
+{% assign metadata = site.pages | where: "output", "false" | where: "componentName", "th-empathy" | first %}
+<p>Sponsor: {{ metadata.sponsor }}</p>
+<p>Accessibility statement: {{ metadata.a11yStatement }}</p>
+<p>Progressive enhancement note:</p>
+<ul>
+    {% for peNote in metadata.peNote %}
+    <li>{{ peNote }}</li>
+    {% endfor %}
+</ul>
+
+<h2>CSS classes</h2>
+<dl class="dl-horizontal">
+    <dt><code>.empathy</code></dt>
+    <dd>Set pruple underline to the <code>&lt;h1&gt;</code> element of the page.</dd>
+    <dt><code>.btn-empathy</code></dt>
+    <dd>Along with <code>.btn-primary</code> or <code>.btn-default</code>, set a purple colour thematic to buttons.</dd>
+</dl>
+
+<h2>Examples</h2>
+<h3>Purple Underline to the <code>&lt;h1&gt;</code> </h3>
+<p>The example has been applied to the <code>&lt;h1&gt;</code> element of this page.</p>
+<h4>Code</h4>
+<pre><code>&lt;h1 property="name" id="wb-cont" dir="ltr" class="empathy"&gt;&lt;-- Page title --&gt;&lt;/h1&gt;</code></pre>
+
+<h3>Purple Button</h3>
+<button class="btn btn-primary btn-empathy" type="button">Button</button>
+<h4>Code</h4>
+<pre><code>&lt;button class="btn btn-primary btn-empathy" type="button"&gt;Button&lt;/button&gt;</code></pre>
+
+<h3>Light Purple Button</h3>
+<button class="btn btn-default btn-empathy" type="button">Button</button>
+<h4>Code</h4>
+<pre><code>&lt;button class="btn btn-default btn-empathy" type="button"&gt;Button&lt;/button&gt;</code></pre>
+
+<h3>Steps Progress</h3>
+<section class="panel panel-default stepsquiz">
```

Comment:
A section must have an heading, move the heading inside the section element

```suggestion
<section class="panel panel-default stepsquiz">
<h3>Steps Progress</h3>
```

or remove the section semantic
```suggestion
<h3>Steps Progress</h3>
<div class="panel panel-default stepsquiz">
```